qdrant-landing/content/documentation/hybrid-cloud/configure-scale-upgrade.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Configure, Scale & Update Clusters
+weight: 3
+---
+
+# Configure, Scale & Update Qdrant Hybrid Cloud Clusters
+
+## Configure Clusters
+
+Next to the Hybrid Cloud specific scheduling options, you can change various advanced configuration options for your clusters. See [Configure Clusters](/documentation/cloud/configure-cluster/) for more details.
```

Comment:
'Alongside Hybrid Cloud specific scheduling options, you can also adjust various other advanced configuration options for your clusters.'